iis-workflows/iis-workflows-top/src/main/resources/eu/dnetlib/iis/workflows/top/preprocessing/main/oozie_app/workflow.xml

```diff
@@ -119,6 +119,12 @@
 			<description>metadata extraction cache location, path pointing to
 				root cache directory holding meta.json file</description>
 		</property>
+		<property>
+            <name>referenceextraction_project_fundingclass_blacklist_regex</name>
+            <value>$^</value>
+            <description>regexp matching funding class, describes projects which 
+            should be blacklisted by reference extraction algorithm</description>
+        </property>
```

Comment:
Right. I see 2 options:

* Finding an option/plugin in Eclipse that automatically makes the indentation entered with tab of the same type (tabs/spaces) as the one used in the surrounding code.
* Showing tabs and spaces in the code explicitly (see e.g. this [SO answer](http://stackoverflow.com/a/27828467/817499)).